intro/mqtt/host-client/Cargo.toml

```diff
@@ -10,4 +10,7 @@ rumqttc = "0.10.0"
 rand = "0.8.4"
 toml-cfg = "0.1"
 get-uuid = { path = "../../../common/lib/get-uuid" }
-mqtt-messages = { path = "../../../common/lib/mqtt-messages" }
\ No newline at end of file
+mqtt-messages = { path = "../../../common/lib/mqtt-messages" }
+
+[patch.crates-io]
+riscv = { git = "https://github.com/rust-embedded/riscv", branch = "master"}
```

Comment:
this should not need this patch, as the host client is built on the host and not for a risk-v target.